stablehlo/reference/Element.cpp

```diff
@@ -120,6 +120,43 @@ Element Element::operator+(const Element &other) const {
   report_fatal_error(std::move(err));
 }
 
+Element Element::operator-(const Element &other) const {
```

Comment:
It seems that most of the element-wise arithmetic operation will share the same code. How about extracting creating a function out of the common code. 

```
Element computeArithmaticOperation(
    const Element &e, std::function<APInt(APInt)> intOperation, std::function<APFloat(APFloat)> floatOperation);
```

Reply:
+1. This is similar to our discussion in #162, so I proposed a pull request that generalizes this to use a single concept. Let's discuss in #190.